eucalyptus.te

```diff
@@ -208,6 +210,9 @@ allow eucalyptus_cluster_t self:netlink_route_socket create_netlink_socket_perms
 allow eucalyptus_cluster_t self:sem create_sem_perms;
 allow eucalyptus_cluster_t self:tcp_socket create_stream_socket_perms;
 allow eucalyptus_cluster_t self:udp_socket create_socket_perms;
+allow eucalyptus_cluster_t self:dir search;
+allow eucalyptus_cluster_t self:file { write open };
```

Comment:
What AVCs and SYSCALLs triggered these?  This smells like a place to use `ps_process_pattern`.